x/shield/keeper/helpers_test.go

```diff
@@ -26,6 +27,12 @@ var (
 	acc4 = sdk.AccAddress(PKS[3].Address().Bytes())
 	acc5 = sdk.AccAddress(PKS[4].Address().Bytes())
 
+	val1 = sdk.ValAddress(PKS[0].Address())
```

Comment:
let's change to `pks` instead of `PKS`?

Reply:
I made the variable but yeah I think pks is better